draft-ietf-rats-architecture.md

```diff
@@ -610,6 +610,14 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the attestation evidence is conveyed. As an example, if the device and Verifier
+communicate over a link that is established using a Root of Trust (e.g. cellular
+communications link where authentication is tied to the presence of a secure element),
```

Comment:
I don't think we should use the term "secure element" in lower case, as it's ambiguous whether we mean Secure Element (per GP) or something more generic

Reply:
Changes to "enclave" as per RFC 4949